Modules/Module_Processing/Module_Population/xASL_stat_GetDICOMStatistics.m

```diff
@@ -98,6 +98,14 @@ function xASL_stat_GetDICOMStatistics(x, ScanType, HasSessions, bOverwrite)
 	% Replace all cells by a string created by concatenating the strings inside
 	TSV{listCells(indexCells)} = strjoin(TSV{listCells(indexCells)}, '/');
 end
+% Find all columns with struct in it
```

Comment:
Would be nicer if we can disentangle the structs to columns, if they are numel(struct)<20 . Because if this includes e.g. multiple PLDs, we want to see them here. This function generates a TSV overview that users can use to compare different subjects to verify that there were no sequence changes along the acquisition.

Ideally this struct unwrapping to string columns happens inside the loop above — i.e. on the single subject level — because the struct can contain different fields for different subjects?

Thinking about it, the input for this function is NOT dicom fields anymore but the BIDS ASL4D.JSONs. So the fields that there are, are well controlled. So it would be helpful to check which JSON field caused the problem for Bea, and ensure that this ends up in this TSV table.

Especially since this is a main QC point, that avoids the user (or us for clinical trials!) going through all the JSONs one-by-one, I do think we should fix this now, even though this may take some time.

Reply:
I see, then issuing a warning is probably best for now? As you say something wrong happened, and it would be good if we can check how this happened (although not too important) and we should not print the structs.